seaborn/categorical.py

```diff
@@ -1438,6 +1440,229 @@ def plot(self, ax):
         if self.orient == "h":
             ax.invert_yaxis()
 
+class _LVPlotter(_CategoricalPlotter):
+
+    def __init__(self, x, y, hue, data, order, hue_order,
+                 orient, color, palette, saturation,
+                 width, k_depth, linewidth, box_widths):
+
+        if width is None:
```

Comment:
These default values should be specified in the function signature for `lvplot`.